### PR TITLE
revise copytext for Social Mobility’s section C

### PR DESCRIPTION
**Pull request by macool**

### CODE REVIEW

File: app/forms/qae_2014_forms/social_mobility/social_mobility_step3.rb [hunk 1]
```diff
@@ -224,7 +186,7 @@ def mobility_step3
           drop_conditional :drops_in_turnover
         end
 
-        textarea :drops_in_turnover, "Explain any drops in turnover, export sales, total net assets and net profits, as well as any losses made." do
+        textarea :drops_in_turnover, "Explain any drops in turnover, total net assets and net profits, as well as any losses made." do
           classes "sub-question js-conditional-drop-question"
           sub_ref "C 4.6"
```
Comment (rusllonrails) [inline app/forms/qae_2014_forms/social_mobility/social_mobility_step3.rb]: @macool you probably need to correct sub_ref to:
```
sub_ref "C 4.4"
```

**Comment by rusllonrails** [comment]
@macool looks good :+1: , but probably need to corrects sub_ref, see comment above ^

**Comment by macool** [comment]
thanks @rusllonrails I've fixed the PR

**Comment by rusllonrails** [comment]
Nice :+1:

**Pull request merged**